File: commons-text/src/main/java/pl/chilldev/commons/text/formatter/HtmlFormatter.java
/*
 * This file is part of the ChillDev-Commons.
 *
 * @license http://mit-license.org/ The MIT license
 * @copyright 2016 © by Rafał Wrzeszcz - Wrzasq.pl.
 */

package pl.chilldev.commons.text.formatter;

/**
 * (X)HTML format handler.
 */
public class HtmlFormatter implements FormatterInterface
{
    /**
     * {@inheritDoc}
     */
    @Override
    public String transform(String text)
    {
        // this is already our desired format
        return text;
    }
}

File: commons-text/src/main/kotlin/pl/wrzasq/commons/text/formatter/HtmlFormatter.kt
/*
 * This file is part of the pl.wrzasq.commons.
 *
 * @license http://mit-license.org/ The MIT license
 * @copyright 2016, 2019, 2021 © by Rafał Wrzeszcz - Wrzasq.pl.
 */

package pl.wrzasq.commons.text.formatter

/**
 * (X)HTML format handler.
 */
class HtmlFormatter : FormatterInterface {
    // this is already our desired format
    override fun transform(text: String): String = text
}
